examples/hello-world/package-lock.json

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
It's surprising that I don't see entries for `@loopback/*` dependencies in the generated `package-lock.json`. Is it by design as they are symbolically linked? If so, what's the impact for published modules?